EXAMPLES.md

```diff
@@ -0,0 +1,60 @@
+# Examples
+
+## In a web page:
+
+Here's an example, showing how the dependency parse for the sentence `"Sam likes bananas"` can be
+rendered using `hierplane`:
```

Comment:
Maybe call out that this is using the CDN link method? Makes sense to me, but I could see it being a bit confusing if the user is also looking at the built-in JSON examples that are formatted/organized slightly differently.

Reply:
I think that's implied by the `In a web page` title, right?  Does that make sense?

Reply:
Works for me. I trust your judgement. :)